source/api/commands/click.md

```diff
@@ -100,27 +100,28 @@ cy.get('button').click(15, 40)
 Forcing a click overrides the {% url 'actionable checks' interacting-with-elements#Forcing %} Cypress applies and will automatically fire the events.
 
 ```javascript
-cy.get('button').click({ force: true })
+cy.get('.close').as('closeBtn')
+cy.get('@closeBtn').click({ force: true })
 ```
 
 ### Force a click with position argument
 
 ```javascript
-cy.get('button').click('bottomLeft', { force: true })
+cy.get('#collapse-sidebar').click('bottomLeft', { force: true })
 ```
 
 ### Force a click with relative coordinates
 
 ```javascript
-cy.get('button').click(5, 60, { force: true })
+cy.get('#footer .next').click(5, 60, { force: true })
 ```
 
-### Click all buttons found on the page
+### Click all elements with id starting with 'btn'
 
 By default, Cypress will error if you're trying to click multiple elements. By passing `{ multiple: true }` Cypress will iteratively apply the click to each element and will also log to the {% url 'Command Log' test-runner#Command-Log %} multiple times.
 
 ```javascript
-cy.get('button').click({ multiple: true })
+cy.get('[id^=btn]').click({ multiple: true })
```

Comment:
nice

Reply:
I would also add an example showing how to click on an element with dynamic CSS style classes like 
```
// clicks on button with dynamic class
// like <div class="abc-save-123">
cy.get('class=*save').click() 
```